Lib/pdb.py

```diff
@@ -477,6 +483,9 @@ def precmd(self, line):
                 next = line[marker+2:].lstrip()
                 self.cmdqueue.append(next)
                 line = line[:marker].rstrip()
+
+        # Replace all the convenience variables
```

Comment:
Instead of Replace, maybe Rename or Move?

Reply:
This is literally a string replacement, to replace the `$foo` literal in command to `__pdb_convenience_variables["foo"]`. Do you think rename or move is more appropriate? The `$` in user command is more like an alias for user to type in.

Reply:
The way it's worded it sounds like replacing the variable, and that's not what you're doing.

Reply:
Maybe "Convert all the convenience variables to their Python format?"

Reply:
Again, this sounds like you're doing something to the variables (converting them, could mean their values). You're just renaming them.

Reply:
This is a comment in the source code, so it's for future developers instead of pdb users. From a developer's view, "convenience variable" never exists. They are just some string alias which would be replaced later with actual Python code.

A fair comparison is MACRO in C. In GNU's [documentation](https://gcc.gnu.org/onlinedocs/cpp/Macros.html):

> Whenever the name is used, it is replaced by the contents of the macro. 

That's exactly what a convenience variable is - a MACRO. Whenever the convenience variable is used in pdb commands, it would be replaced by it's actual Python representation. It's literally `#define $(name) ____pdb_convenience_variables["#name"]` (yes that's not a valid C macro, just to make the point).

Do you think "Replace(Convert) the convenience variable representation with(to) the corresponding Python representation" is better? It would reduce the ambiguity for whether the variable is the variable itself or the value it holds.

Reply:
We're overthinking it, leave it as it is.